test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -316,7 +316,7 @@ def test_pytorch_training_torchdata_gpu(
     _, image_framework_version = get_framework_and_version_from_tag(pytorch_training)
     if test_utils.is_image_incompatible_with_instance_type(pytorch_training, ec2_instance_type):
         pytest.skip(f"Image {pytorch_training} is incompatible with instance type {ec2_instance_type}")
-    if Version(image_framework_version) in SpecifierSet("==1.11.*"):
+    if any(Version(image_framework_version) in v for v in [SpecifierSet("==1.11.*"), SpecifierSet("==1.12.*")]):
```

Comment:
You can actually use:
```suggestion
if Version(image_framework_version) in SpecifierSet(">=1.11,<1.13")
```
instead.

Reply:
Ack'ed below.